src/utils.tsx

```diff
@@ -566,6 +566,12 @@ export function parseErrorResponse(errorMessage: string): string {
     returnMessage = 'Content type not supported.';
   } else if (errorMessage.includes('not a valid image')) {
     returnMessage = 'File given is not a valid image.';
+  } else if (
+    errorMessage.includes('file name already exists within the parent entity.')
+  ) {
+    returnMessage =
+      'A file with this name already exists. To rename it, remove the file and add it again. You’ll then be able to click the pencil icon just below the file to change its name.';
```

Comment:
```suggestion
    returnMessage =
      'A file with this name already exists. To rename your file: remove it, add it back and click the edit icon below the file to change its name.';
```

Very minor suggestion, but I think wording it like this makes it clearer which file you're talking about, and makes it sound more like an error message, if that makes sense.

Would need to be updated in other test/implementation files as well, if changed.